pulp_rpm/app/models/modulemd.py

```diff
@@ -21,6 +21,11 @@ class Modulemd(Content):
             The modulemd's stream
         version (Text):
             The version of the modulemd.
+        static_context (Boolean):
+            If True, then he context flag is a string of up to thirteen [a-zA-Z0-9_] characters
```

Comment:
s/he/the/

Reply:
Maybe we do not need `flag`? just `context`? For some reason, it confused me, since I expect flag to be boolean. Maybe I'm wrong. Up to you.

Reply:
Yeah, I was confused by that as well, and went through the exact same set of steps you did.

Reply:
fixed the comment, good eye!